packages/eslint-plugin-wpcalypso/docs/rules/i18n-no-this-translate.md

```diff
@@ -0,0 +1,31 @@
+# Disallow this.translate()
+Disallows the usage of `this.translate()` in favor of the high order component `localize()` from `i18n-calypso` repository.
+
+## Rule Details
+
+This rules aims to prevent the usage of `this.translate` that uses the mixin approach which makes the components harder to test.
+The following is considered a warning:
+```js
+const MyComponent = React.createClass( {
+	render() {
+		return this.translate( 'Hello World' );
```

Comment:
`this.props.translate`

Reply:
The `createClass` is already gone. The `this.translate` is showing a lint error from the mixin days.